www/article.html

```diff
@@ -2,7 +2,7 @@
 <html>
     <head>
         <meta charset="utf-8">
-        <title>Placeholder for injecting an article into the iframe</title>
+        <meta name="description" content="Placeholder for injecting an article into the iframe or window">
```

Comment:
We need to avoid setting a title in the dummy article, because it sometimes sticks and doesn't get replaced by the article title when loading a new tab or window. However, we still need a string in the html that allows us to identify that the dummy article is loaded (see in code below where we search for this string).